base/src/main/java/io/spine/validate/MessageFieldValidator.java

```diff
@@ -47,25 +47,26 @@
      *         if {@code true} the validator would assume that the field is required even if
      *         such constraint is not explicitly set
      */
-    @SuppressWarnings("unchecked"/*`When` option validates `Timestamp`s, which extend `Message`.*/)
     MessageFieldValidator(FieldValue<V> fieldValue, boolean assumeRequired) {
-        super(fieldValue,
-              assumeRequired,
-              ImmutableSet.of(((FieldValidatingOption<?, V>) When.create())));
+        super(fieldValue, assumeRequired, additionalOptions());
     }
 
     @Override
-    protected void validateOwnRules() {
+    protected List<ConstraintViolation> validate() {
```

Comment:
Please use `ImmutableList` in results when we do so. It would minimize the need for explaining the fact in Javadocs.